develop/start-develop.sh

```diff
@@ -3,5 +3,5 @@ set -x
 set -e
 cd $SOURCE_DIR
 npm install
-node_modules/bower/bin/bower install --allow-root
-node_modules/grunt-cli/bin/grunt serve --allow-root
+bower install --allow-root
```

Comment:
A while ago I've learned that node executables create a link in `node_modules/bin/` so no need to install them globally.